images/installer/Dockerfile.upi.ci

```diff
@@ -23,6 +23,8 @@ COPY --from=builder /go/src/github.com/openshift/installer/upi /var/lib/openshif
 
 COPY --from=govc /govc /bin/govc
 
+# RUN curl https://packages.microsoft.com/config/rhel/8/prod.repo | tee /etc/yum.repos.d/microsoft.repo
+
```

Comment:
The prod repo is already added below. Can you remove these lines?

Reply:
Yes.  Good catch.  I can update this.